tools/StaticAnalysis/Exceptions/Az.ServiceFabric/SignatureIssues.csv

```diff
@@ -6,3 +6,8 @@
 "Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll","Microsoft.Azure.Commands.ServiceFabric.Commands.RemoveAzureRmServiceFabricSetting","Remove-AzServiceFabricSetting","1","8510","Cmdlet 'Remove-AzServiceFabricSetting' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
 "Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll","Microsoft.Azure.Commands.ServiceFabric.Commands.SetAzureRmServiceFabricSetting","Set-AzServiceFabricSetting","1","8510","Cmdlet 'Set-AzServiceFabricSetting' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
 "Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll","Microsoft.Azure.Commands.ServiceFabric.Commands.SetAzureRmServiceFabricUpgradeType","Set-AzServiceFabricUpgradeType","1","8510","Cmdlet 'Set-AzServiceFabricUpgradeType' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
+"Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll","Microsoft.Azure.Commands.ServiceFabric.Commands.NewAzServiceFabricService","New-AzServiceFabricService","1","8410","Parameter Stateless of cmdlet New-AzServiceFabricService does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll","Microsoft.Azure.Commands.ServiceFabric.Commands.UpdateAzServiceFabricApplication","Update-AzServiceFabricApplication","1","8410","Parameter MinimumNodes of cmdlet Update-AzServiceFabricApplication does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll","Microsoft.Azure.Commands.ServiceFabric.Commands.UpdateAzServiceFabricApplication","Update-AzServiceFabricApplication","1","8410","Parameter MaximumNodes of cmdlet Update-AzServiceFabricApplication does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.dll","Microsoft.Azure.Commands.ServiceFabric.Commands.UpdateAzServiceFabricApplication","Update-AzServiceFabricApplication","1","8410","Parameter DefaultServiceTypeMaxPercentUnhealthyServices of cmdlet Update-AzServiceFabricApplication does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
I believe some of these parameters need rather to be renamed in signular than treated as exceptions. @markcowl what's your opinion?